core-tests/jvm/src/test/scala-2.12/zio/StacktracesSpec.scala

```diff
@@ -293,6 +295,73 @@ object StackTracesSpec extends DefaultRunnableSpec {
         assert(cause.traces.head.stackTrace.size)(equalTo(3)) &&
         assert(cause.traces.head.stackTrace.exists(_.prettyPrint.contains("selectHumans")))(isTrue)
       }
+    },
+    testM("basic option test") {
```

Comment:
Can we move these from `StackTracesSpec` to `ZIOSpec`? This file should only be for tests of the tracing functionality which shouldn't be impacted here or by your prior PR.

Reply:
sure, I was not sure where to add those scenarios, but indeed the file name of the test was not matching pretty well. I will change it in this PR